pkg/utils/syscalls/syscalls_amd64.go

```diff
@@ -0,0 +1,741 @@
+//go:build linux && amd64
```

Comment:
I think this build constraint is by default, as we saw in the official doc https://pkg.go.dev/cmd/go#hdr-Build_constraints

Reply:
yes please, since they bring no value

Reply:
Done